### ISSUE TITLE
[Bug]: Discount sidebar nudges are not displayed

### ISSUE THREAD

**Issue by dsas** [Automattic/wp-calypso#59177]
### Quick summary

If there was a currently active discount applicable to the current user, it was displayed in the sidebar just below the domain. This isn't working.

There's some [documentation on how this active discount feature is expected to work](https://martechp2.wordpress.com/2020/11/23/some-calypso-promo-nudge-documentation/) from last November.

It appears to have been broken accidentally in April 2020 (#40723), and no-one has noticed it. Is it worth fixing - would it now be unwelcome or should the buggy code just be removed? I can't get any data in tracks showing how effective it was because it's not worked for so long.

The problem is that [`discountOrFreeToPaid`](https://github.com/Automattic/wp-calypso/blob/e64ec136821355ae29226b39f0ab409f55ef64b3/client/my-sites/current-site/notice.jsx#L120) is a HTML string containing a nudge prompting people to claim a discount. however the only place it's used is in this [boolean expression](https://github.com/Automattic/wp-calypso/blob/e64ec136821355ae29226b39f0ab409f55ef64b3/client/my-sites/current-site/notice.jsx#L123-L124).

### Steps to reproduce

1. Open calypso and go to a site

To be really sure it's related to the sidebar rather than an API etc: 
 1. In your calypso checkout, open client/my-sites-current-site/notice.jsx
 2. In the `connect` method, change the `activeDiscount` line to hardcode `activeDiscount: {nudgeText: 'foo', nudgeEndsTodayText: 'bar', ctaText: 'baz', name: 'quark'},`

### What you expected to happen

A discount to be displayed in the left hand sidebar with a button saying 'baz', like this:

![image](https://user-images.githubusercontent.com/93301/145997750-cf08591e-6c9f-4f7b-bddd-9bd4baf1487b.png)

### What actually happened

A discount was not displayed - a JITM nudge was displayed though, which is different.

### Context

Found when deleting extranous code in #59058

### Reproducibility

Consistent

### Severity

All

### Available workarounds?

No but the platform is still usable

**Comment by cuemarie** [comment]
_This report was found during a recent backlog sweep._

Hey @dsas ! I'm not familiar with discount nudges in the last couple of years - do we still run these?  If not, I think we can close this as no longer relevant, but wanted to double check with you first. Thanks!

📌 ACTIONS
- Needs author reply

**Comment by dsas** [comment]
Hey @cuemarie, wow I don't even remember looking at this.

Looks like I reported it directly to Martech when I filed this bug: pau2Xa-3J5-p2 and got no response. Looking at the list of discounts they all expired at least four years ago. According to my initial report it's been broken since 2020.

So there's no user-impacting bug and it looks like no-one cares about the feature. There's some useless & buggy code that maybe Martech want to clean-up but there's no need to track as a bug any more.
